server/route.go

```diff
@@ -143,16 +143,20 @@ func (c *client) processRouteInfo(info *Info) {
 		// Send our local subscriptions to this route.
 		s.sendLocalSubsToRoute(c)
 		if sendInfo {
-			// Need to get the remote IP address.
-			c.mu.Lock()
-			switch conn := c.nc.(type) {
-			case *net.TCPConn, *tls.Conn:
-				addr := conn.RemoteAddr().(*net.TCPAddr)
-				info.IP = fmt.Sprintf("nats-route://%s/", net.JoinHostPort(addr.IP.String(), strconv.Itoa(info.Port)))
-			default:
-				info.IP = c.route.url.String()
+			// If IP isn't already set on info
+			if info.IP == "" {
```

Comment:
Should these options be handled reload?  (I don't think this would be correct if this were implemented there).

Reply:
@ColinSullivan1 per @tylertreat I still need to go through and add reload support. Will correctly reload this field.